app/controllers/bids_controller.rb

```diff
@@ -31,13 +30,14 @@ def new
   end
 
   def confirm
-    @auction = Presenter::Auction.new(Auction.find(params[:auction_id]))
-    @bid = Presenter::Bid.new(PlaceBid.new(params, current_user).dry_run)
+    @auction = Policy::Auction.factory(Auction.find(params[:auction_id]), current_user)
+    @bid = Policy::Bid.new(PlaceBid.new(@auction, params).dry_run, @auction)
```

Comment:
Indeed. This was just a way to dry-run the user bid for a Confirmation, but it's a bit weird now